lib/tests/misc.nix

```
@@ -301,10 +299,11 @@ runTests {
 
   testMakeIncludePathWithPkgs = {
     expr = (makeIncludePath [
-      pkgs.openssl
-      pkgs.zlib
+      { dev = "/nix/store/bash"; }
+      # lib.getOutput "dev" return out if "dev" is not found
+      { out = "/nix/store/openssl"; }
```

```suggestion
      { dev = { outPath = "/nix/store/bash"; }; }
      # lib.getOutput "dev" return out if "dev" is not found
      { out = { outPath = "/nix/store/openssl"; }; }
```

this is more realistic, since `out` and `dev` will be almost-identical attrsets with the `outputName` and `outPath` fields changed.